.github/actions/spelling/expect.txt

```diff
@@ -0,0 +1,235 @@
+abi
+adt
+Alletra
+alloc
+alltgpt
+alua
+aptpl
+ascq
+ata
+autoconfig
+autodetected
+barbie
+BINDIR
+blkid
+bmarzins
+cciss
+CFLAGS
+CLARiiON
+christophe
+clangd
+clariion
+cmdline
+cmocka
+coldplug
+commandline
+COMPAQ
+configdir
+configfile
+configurator
+cplusplus
+CPPFLAGS
+ctx
+dasd
+datacore
+ddf
+Debian
+DESTDIR
+devmaps
+devname
+devnode
+devpath
+DEVTYPE
+DGC
+DIO
+directio
+disablequeueing
+dmevent
+dmmp
+dmraid
+dmsetup
+dracut
+EAGAIN
+emc
+Engenio
+EVPD
+failback
+failover
+fds
+FFFFFFFF
+fge
+followover
+forcequeueing
+fpin
+fulldescr
+getprkey
+getprstatus
+getuid
+github
+GPLv
+GPT
+HBAs
+hbtl
+hds
+hotplug
+HPE
+HSG
+HSV
+Huawei
+hwhandler
+hwtable
+iet
+ifdef
+ifndef
+igroup
+img
+inttypes
+iscsi
+isw
+kpartx
+LDFLAGS
+len
+libaio
+LIBDEPS
+libdevmapper
+libdmmp
+libedit
+libjson
+libmpathpersist
+libmpathvalid
+libmultipath
+libreadline
+libsystemd
+libudev
+libudevdir
+liburcu
+linux
+lpthread
+Lun
+lvm
+Marzinski
+mpath
+mpathb
+mpathpersist
+mpathvalid
+msecs
+multipathc
+multipathd
+multipathed
+multipathing
+multipaths
+multiqueue
+mwilck
+NOLOG
+nompath
+NOSCAN
+Nosync
+nvme
+OBJDEPS
+ontap
+OOM
+opensvc
+OPTFLAGS
+paramp
+partx
+pathgroup
+pgpolicy
+plugindir
+PNR
+prefixdir
+preferredip
+preferredsds
+prgeneration
+Primera
+prioritizer
+prkeys
+PROUT
+pthreads
+rdac
+rdma
+readcap
+readdescr
+readfd
+readkeys
+READLINE
+readsector
+rebranded
+reconfig
+redhat
+restorequeueing
+retrigger
+rhabarber
+rootprefix
+rootprefixdir
+rport
+rtpi
+SANtricity
```

Comment:
Similarly, this is parsed as `Ntricity` -- the `SA` is split off and then ignored because it's too short.